content/rest-api/versions/v1.0/index.textile

```diff
@@ -141,18 +141,18 @@ Wherever possible, success response bodies contain links, in "HATEOS":http://en.
 
 h3(#versioning). Versioning
 
-By default, all requests receive the latest version of the API, which is currently @1.0@.
+By default, all requests receive the latest version of the API, which is currently @1.1@.
```

Comment:
Did you mean to modify this file as it's in a `versions` folder?

Reply:
Good spot, no.